AsyncDisplayKit/ASTextNode.h

```diff
@@ -71,6 +91,12 @@ typedef NS_ENUM(NSUInteger, ASTextNodeHighlightStyle) {
 @property (nonatomic, assign) NSUInteger maximumNumberOfLines;
 
 /**
+ @abstract The vertical alignment of text within the receiver.
+ @discussion Defaults to ASTextNodeVerticalTextAlignmentTop
+ */
+@property (nonatomic, assign) ASTextNodeVerticalTextAlignment verticalTextAlignment;
```

Comment:
Let's move this to be the very last property.  Add a #pragma mark - Miscellaneous before it just to separate it from Placeholders (not so sure those pragmas are useful in the first place, but we'll leave them for now).